docs/alerts-and-notifications/notifications/centralized-cloud-notifications/manage-notification-methods.md

```diff
@@ -53,19 +53,20 @@ To manage Space notification settings, you will need the following:
 To manage user-specific notification settings, you will need the following:
 
 - A Cloud account
-- Access to, at least, a Space
```

Comment:
This is an interesting case! Usually, we see missing commas, but here we have the opposite problem